README.md

```diff
@@ -21,6 +21,10 @@ make uninstall
 ```
 Any of these commands may require root privileges depending on your environment.
 
+To cleanup credentials located at root (you should run after uninstall):
```

Comment:
Last nitpick: you don't need to run this after install as it does the uninstall automatically, so maybe say to run this instead if you want to purge config files as well.